src/components/Shared/GridToolbar.js

```diff
@@ -86,6 +86,7 @@ const GridToolbar = ({
   function clear() {
     setSearchValue('')
     onSearch('')
+    onSearchClear()
```

Comment:
why?

Reply:
i need  onSearchClear to  clear filter and it is present since 3 month in history